### PR TITLE
Fix dotnet-inproc-appservice build

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

File: host/4/bullseye/python/python37/python37-composite.template [hunk 1]
```diff
@@ -62,8 +62,6 @@ COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
 
 ENV FUNCTIONS_WORKER_RUNTIME_VERSION=3.7
 
-EXPOSE 2222 80
```
Comment (CooperLink) [inline host/4/bullseye/python/python37/python37-composite.template:65]: if we remove these expose rules then SSH won't work on Python3.7

File: host/4/bullseye/python/python38/python38-composite.template [hunk 2]
```diff
@@ -61,8 +61,6 @@ COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
 
 ENV FUNCTIONS_WORKER_RUNTIME_VERSION=3.8
 
-EXPOSE 2222 80
```
Comment (CooperLink) [inline host/4/bullseye/python/python38/python38-composite.template:64]: Removing expose removes ssh

File: host/4/mariner/dotnet-isolated/dotnet7-isolated-mariner.Dockerfile [hunk 3]
```diff
@@ -37,5 +38,6 @@ RUN dnf install -y glibc-devel
 
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=aspnet7 [ "/usr/share/dotnet", "/usr/share/dotnet" ]
+COPY --from=dn8-sdk-image [ "/usr/share/dotnet", "/usr/share/dotnet" ]
```
Comment (CooperLink) [inline host/4/mariner/dotnet-isolated/dotnet7-isolated-mariner.Dockerfile:41]: Why do we need to include the dotnet8 sdk image

Comment (pragnagopa) [reply host/4/mariner/dotnet-isolated/dotnet7-isolated-mariner.Dockerfile:41]: without this running the docker image is failing. I merged the PR as the changes are close to green build. @CooperLink - I think its good for you take a final pass after I get all the green builds to see if there is any other fixes we need to do!

**Pull request merged**